part-2-server/server.js

```diff
@@ -0,0 +1,42 @@
+var http = require('http');
+
+var fs = require('fs');
+
+var server = http.createServer(function(req, res){
+	var url = req.url;
+	if(url==='/'){
+		fs.readFile('index.html',function(err, data){
+			if(err) throw err;
+			else{
+				res.write(data.toString());
+				res.end();
+			}
+		})
+		
+	}
+	else if(url==='/page-2'){
+		fs.readFile('page-2.html',function(err, data){
+			if(err) throw err;
+			else{
+				res.write(data.toString());
+				res.end();
+		}
+	})
+	}
+	else {
```

Comment:
This works but is a little inefficient because you need to run a for loop during every single call to 'poem-'. Could avoid looping by splitting the url into substrings